app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -48,6 +48,12 @@ def monitor_events
   end
 
   def process_event(event)
+    event_data = extract_event_data(event)
```

Comment:
What if the new function returns nil in line 73 and event data is nil?
The old code would not add_queue and log in that case